openflow_input/bsn_tlv

```diff
@@ -1393,3 +1393,8 @@ struct of_bsn_tlv_lossless : of_bsn_tlv {
     uint16_t type == 188;
     uint16_t length;
 };
+
+struct of_bsn_tlv_redundant_mgmt : of_bsn_tlv {
+    uint16_t type == 189;
+    uint16_t length;
+};
```

Comment:
Is this a boolean or we need to pass the port name?

Reply:
This is a boolean. It is meant to be associated with a port key in the port_config gentable.

Reply:
Makes sense.